build.sbt

```diff
@@ -75,4 +77,15 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
   coverageFailOnMinimum := true,
   coverageHighlighting := true,
   parallelExecution in Test := false
-)
\ No newline at end of file
+)
+
+lazy val silencerSettings: Seq[Setting[_]] = {
```

Comment:
Are these the settings we need @PatrykRudnicki to get rid of our warnings?

Reply:
I've used them to silence dozens of warning coming from twirl templates and `.routes` files.

Reply:
Yes, it will reduce amount of warnings